.github/workflows/purge-azure-test-resources.yaml

```diff
@@ -91,9 +91,36 @@ jobs:
               az group delete --resource-group $line --yes --verbose --no-wait
           done
 
+  purge_bicep_types:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.BICEPTYPES_CLIENT_ID }}
+          tenant-id: ${{ secrets.BICEPTYPES_TENANT_ID }}
+          subscription-id: ${{ secrets.BICEPTYPES_SUBSCRIPTION_ID }}
+
+      # Delete Bicep types that are older than 90 days from the test registry 
```

Comment:
Curious why keep them around for 90 days? Are they reused across PRs?

Reply:
90 days was an arbitrary pick but it seemed like the safest bet since bigger PRs take longer to review. Technically the types get overwritten every time the tests run so we can change this value

Reply:
synced offline, updating this to 30 days